gramps/gen/utils/grampslocale.py

```diff
@@ -525,6 +525,8 @@ def __init_first_instance(self):
         # with locale instead of gettext. Win32 doesn't support bindtextdomain.
         if self.localedir:
             if not sys.platform == 'win32':
+                # bug12278, _build_popup_ui() under linux and macOS
+                locale.textdomain(self.localedomain)
```

Comment:
@romjerome Don't do this, it will make a conflict when @prculley next merges-up gramps51. That merge-up will take care of bug 12278 in master.